scripts/devenv-builder/configure-vm.sh

```diff
@@ -80,17 +95,20 @@ sudo firewall-cmd --permanent --zone=trusted --add-source=10.42.0.0/16
 sudo firewall-cmd --permanent --zone=trusted --add-source=169.254.169.1
 sudo firewall-cmd --reload
 
-# Run MicroShift Executable > Configuring MicroShift
```

Comment:
The modification of firewall rules should not be performed on a build host and should also be conditional on BUILD_AND_INSTALL.